db-engine/src/main/java/com/splicemachine/db/iapi/types/DataTypeDescriptor.java

```diff
@@ -1602,6 +1602,10 @@ public String toString(){
         return typeDescriptor.toString();
     }
 
+    public String toSparkString(){
```

Comment:
In TypeDescriptorImpl.java you had the opening `{` on a separate line. Be consistent.